robottelo/constants/__init__.py

```diff
@@ -203,6 +203,7 @@ class Colored(Box):
 
 RPM_TO_UPLOAD = "which-2.19-6.el6.x86_64.rpm"
 SRPM_TO_UPLOAD = "which-2.19-6.el6.src.rpm"
+PYTHON_LXML = 'python3-lxml-4.2.3-4.el8.x86_64.rpm'
```

Comment:
Will the `python3-lxml` of this version be present in the repository forever? If we point to the latest AppStream, it would certainly be upgraded to a higher version and this version will not be present in the repo anymore.